### PR TITLE
Add greenlearning

### PR DESCRIPTION
**Pull request by NBoulle**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/greenlearning/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/NBoulle/greenlearning/archive/v{{ version }}.tar.gz"
+  sha256: 7d7d53d5d02d2123add2a58a04efebb6103bab1f2203fda2b287507fa50c5caa
+
+build:
+  number: 0
+  skip: true  # [win]
```
Comment (BastianZim) [inline recipes/greenlearning/meta.yaml:14]: If the only reason you're restricting Windows is a missing dependency, you can still keep it [noarch](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python) and just ignore that. During installation, the solver will just inform the user of the missing dependency.

Comment (NBoulle) [reply recipes/greenlearning/meta.yaml:14]: The build fails on windows due to a missing dependency, so I can ignore this issue and keep it noarch right ?

Comment (BastianZim) [reply recipes/greenlearning/meta.yaml:14]: Yes, that's fine.

Comment (NBoulle) [reply recipes/greenlearning/meta.yaml]: It should be working now.

Comment (BastianZim) [reply recipes/greenlearning/meta.yaml]: Awesome, if you're now ready for review you can ping the team as outlined [here](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team) or let me know if you have any questions.

**Comment by NBoulle** [comment]
@conda-forge-admin, please ping team

File: recipes/greenlearning/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - matplotlib
```
Comment (xhochy) [inline recipes/greenlearning/meta.yaml:23]: Normally `-base` is sufficient. `matplotlib` is only needed if the Qt backend is a hard requirement.
```suggestion
    - matplotlib-base
```

**Pull request merged**